spec/helpers/script_helper_spec.rb

```diff
@@ -43,9 +43,9 @@
           'script[type="application/json"][data-asset-map]',
           visible: :all,
           text: {
-            'clock.svg' => '/clock.svg',
+            'clock.svg' => 'http://test.host/clock.svg',
```

Comment:
This is due to `request` now existing when the function is called in this spec here: https://github.com/18F/identity-idp/blob/f2bc0300432fb66175425bba8acbff39657f1480/app/helpers/script_helper.rb#L78-L80

I'm not sure why it would have changed, but I don't think it affects the primary motivation behind the test?